draft-packages/form/KaizenDraft/Form/Primitives/FieldMessage/FieldMessage.tsx

```diff
@@ -1,59 +1,56 @@
-import classnames from "classnames"
 import * as React from "react"
-import exclamationIcon from "@kaizen/component-library/icons/exclamation.icon.svg"
-
 import { Icon, Paragraph } from "@kaizen/component-library"
+import exclamationWhiteIcon from "@kaizen/component-library/icons/exclamation-white.icon.svg"
+import classnames from "classnames"
 import styles from "./styles.scss"
 
-export type FieldMessageStatus = "default" | "success" | "error"
+export type FieldMessageStatus = "default" | "success" | "error" | "caution"
+
 export type FieldMessageProps = {
   id?: string
   automationId?: string
-  message?: React.ReactNode
+  message?: string | React.ReactNode
   status?: FieldMessageStatus
   reversed?: boolean
   position?: "top" | "bottom"
 }
 
-type FieldMessage = React.SFC<FieldMessageProps>
-
-const warningIcon = (
+const WarningIcon: React.FunctionComponent = () => (
   <span className={styles.warningIcon}>
     <Icon
-      icon={exclamationIcon}
+      icon={exclamationWhiteIcon}
       title="Error message"
       role="img"
       inheritSize={false}
     />
   </span>
 )
 
-const FieldMessage: FieldMessage = ({
+const FieldMessage: React.FunctionComponent<FieldMessageProps> = ({
   id,
   automationId,
   message,
   status = "default",
   reversed = false,
   position = "bottom",
 }) => {
-  const textColor = reversed
-    ? status === "error"
-      ? "dark-reduced-opacity"
-      : "white-reduced-opacity"
-    : "dark-reduced-opacity"
+  const textColor =
```

Comment:
[Refactoring opportunity]
This nested ternary already existed, but it would be great to clean up and make clearer - my 🧠 can't make sense of this ;)
We should also consider adding the ESLint plugin that helps us avoid them in the future.

Reply:
Might create a quick little ticket to address this after :)